testnet/stacks-node/src/tests/integrations.rs

```diff
@@ -49,6 +49,8 @@ const CALL_READ_CONTRACT: &'static str = "
     (ok (contract-call? .other f2 u5)))
 ";
 
+// TODO(gregorycoppola) Add "burn-block-info" section to this test, once a way to configure the
+// Mocknet default Stacks epoch has been decided.
```

Comment:
Specific to this test, can you verify that `burn-block-info` will continue to work even if there was no Stacks block chosen at that height?  Also, verify that we get `(none)` for any block height prior to the first burnchain block height.